<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>docpatop</class>
 <widget class="QDialog" name="docpatop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_sumit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>430</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>561</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_fill">
      <property name="text">
       <string>Please fill your details</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Full Name</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_name</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_phone">
        <property name="text">
         <string>Phone Number</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_phone</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_phone"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_gender">
        <property name="text">
         <string>Gender</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_gender</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_gender"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_email">
        <property name="text">
         <string>Email Id</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_email</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_email"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_state">
        <property name="text">
         <string>State</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_state</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_state"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_city">
        <property name="text">
         <string>City</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_city</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEdit_city"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_aadhar">
        <property name="text">
         <string>Aadhar Card Number</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_aadhar</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_aadhar"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_mjrdis">
        <property name="text">
         <string>Allergies/Major Diseases</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_mjrdis</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEdit_mjrdis"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Specialisation</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QComboBox" name="comboBox_spec">
        <item>
         <property name="text">
          <string>Ayurveda</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cardiology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cosmetology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dental</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dermatology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Endocrinology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ENT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Family Medicine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>General Medicine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>General Physician</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>General Surgery</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Geriatrics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gynaecology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hepatology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Homeopathy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nephrology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Neurology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ophthalmology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Orthopedics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pathology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Physiotherapy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Plastic &amp; Cosmetic Surgery</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rheumatology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Trichology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Urology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Urosurgery</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Registration Number</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_regnum"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_regis">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>371</width>
     <height>561</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
